### PR TITLE
Reduce spacing in the three largest headers

### LINKED ISSUE #537: Too-large header spacing on master branch
Hi all,

I'm using minima on the [ScheduleBot Blog](https://blog.schedulebot.app) and recently switched:

|               | **Old**                  | **New**                       |
|---------------|--------------------------|-------------------------------|
| `Gemfile`     | `gem "minima", "~> 2.5"` | `gem "jekyll-remote-theme"`   |
| `_config.yml` | `theme: minima`          | `remote_theme: jekyll/minima` |

so that I could use the new `custom-head.html` feature.

There's now lots of extra spacing above headers. For example, here are some screenshots (from [this post](https://blog.schedulebot.app/schedulebot/scheduling/2020/09/01/employee-work-scheduling-constraints-shouldn-t-be-this-hard.html) if you need a live example):

## Old Appearance

image

## New Appearance

image

## Question

- Is this change intentional, or a bug?
- If it's intentional, what's the preferred approach to override it?

### PR DESCRIPTION
**Pull request by getaaron**
Closes #537.

## Before (too much spacing)

image

## After

image

### CODE REVIEW

**Comment by ashmaroli** [comment]
I like [what I'm seeing](https://deploy-preview-538--jekyll-minima.netlify.app/junk/2016/05/20/this-post-demonstrates-post-content-styles.html) from this change @getaaron 
Could you add a set of `before` and `after` screenshots to immediately demonstrate the change?

Thanks.

**Comment by MichaelCurrin** [comment]
Agree. Would like to see screenshots.

**Review by mattr- — COMMENTED** [review]
I'm happy with the style change. However, we have our own tooling for managing History.markdown and I'd appreciate it if you would undo the change to that file in this PR.

File: History.markdown [hunk 1]
```diff
@@ -28,6 +28,7 @@
   * Improve the readablility of blockquotes (#481)
   * Extract more color rules into skins (#452)
   * Introduce &#34;Dark Minima&#34; skin (#335)
+  * Adjust header spacing (#537)
```
Comment (getaaron) [inline History.markdown:34]:
```suggestion
```

**Comment by getaaron** [comment]
@mattr- I made your proposed changes to the history file and included screenshots in the PR description.

**Comment by mattr-** [comment]
@getaaron Thanks! I'm leaving on vacation shortly, but have queued this for review after I'm back. Please ping me again on this PR if you don't hear back by the 2022-08-22.
image

**Review by mattr- — APPROVED** [review]

**Comment by mattr-** [comment]
@jekyllbot: merge +minor

**Pull request merged**